PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/resources/META-INF/ruleConfigs.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I think we should design the config file that is extendable and applicable to all rulesets to some degree.
Let's do it in a follow up PR:
But we could probably use values like:

```json
RuleName :{
   "antipattern": "{value}"
   "recommendation": "{value}"
}
```
what do you think?

Reply:
I agree, but in instances where we have a "this detected, use this instead" we can just have one entry for the problem message

Reply:
we all chatted Tuesday morning.  Njeri is going to check in as-is with a github issue filed to track updating the config file format in a separate follow up PR in a few days.